apps/settings/elements/root.html

```diff
@@ -190,25 +184,25 @@ <h2 data-l10n-id="storage">Storage</h2>
       </header>
       <ul>
         <li class="pack-split usb-item">
-          <label class="pack-switch">
-            <input class="usb-switch" data-ignore type="checkbox" disabled>
+          <label class="pack-switch" role="presentation">
```

Comment:
Here I want to drop label semantics since it's unnecessary and confusing. presentation role tells the screen reader to just treat the element as a generic container with not additional semantics (but not remote its sub-tree from being visible, like aria-hidden="true" does). so without it screen reader would say something like: `USB Storage checked check button label`
and with the role presentation it will say: `USB Storage checked check button`
which is more clear to the user